src/field_theory/normal_closure.lean

```diff
@@ -0,0 +1,108 @@
+/-
+Copyright (c) 2023 Thomas Browning. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Thomas Browning
+-/
+
+import field_theory.normal
+
+/-!
+# Normal closures
+
+In this file we define the normal closure of an `intermediate_field`.
+If `K` is an intermediate field of `L/F` (in mathlib this means that `K` is both a subfield of `L`
+and an `F`-subalgebra of `L`), then `K.normal_closure` is the smallest intermediate field of `L/F`
+that contains the image of every `F`-algebra embedding `K →ₐ[F] L`. This will agree with the
+absolute normal closure (`normal_closure` in the root namespace) whenever `L/F` is normal, but with
```

Comment:
Can you add a reference to the declaration saying that this agrees with `normal_closure` if `L/F` is normal?